moco-switchover-downtime-monitor/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/spf13/pflag"
+	"go.uber.org/zap"
+)
+
+const httpServerPort = 8080
+
+var flagNamespace = pflag.String("namespace", "", "The namespace this program runs in. default: read from serviceaccount file")
+
+var currentNamespace = "default"
+
+func newZapLogger() *zap.Logger {
+	logger, err := zap.NewProduction()
+	if err != nil {
+		panic(err)
+	}
+	return logger
+}
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := newZapLogger()
+	defer logger.Sync()
+
+	pflag.Parse()
+	args := pflag.Args()
+
+	clusterNames := args
+
+	if len(clusterNames) == 0 {
+		logger.Error("no MySQLCluster is specified")
+		return 1
+	}
+
+	if *flagNamespace != "" {
+		currentNamespace = *flagNamespace
+	} else {
+		ns, err := os.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+		if err != nil {
+			logger.Error("could not read namespace file", zap.Error(err))
+			return 1
+		}
```

Comment:
I don't want to do so because it makes wastefully complex to exec `kubectl`.
```
    if namespace == "" {
        execCmd("kubectl", "get", "pod", podName)
    } else {
        execCmd("kubectl", "get", "pod", "-n", "namespace", podName)
    }
```